python/pyspark/sql/tests/test_pandas_cogrouped_map.py

```diff
@@ -180,59 +176,26 @@ def left_assign_key(key, l, _):
         assert_frame_equal(expected, result, check_column_type=_check_column_type)
 
     def test_wrong_return_type(self):
-        with QuietTest(self.sc):
-            with self.assertRaisesRegexp(
-                    NotImplementedError,
-                    'Invalid returnType.*cogrouped map Pandas UDF.*MapType'):
-                pandas_udf(
-                    lambda l, r: l,
-                    'id long, v map<int, int>',
-                    PandasUDFType.COGROUPED_MAP)
-
-    def test_wrong_args(self):
         # Test that we get a sensible exception invalid values passed to apply
         left = self.data1
         right = self.data2
         with QuietTest(self.sc):
-            # Function rather than a udf
```

Comment:
`groupby.cogroup.applyInPandas` now always sets the right eval type internally. So we don't have to worry about mis-typed UDF.